android/src/main/java/com/alpha0010/fs/FileAccessModule.kt

```diff
@@ -400,6 +400,26 @@ class FileAccessModule internal constructor(context: ReactApplicationContext) :
     }
   }
 
+  @ReactMethod
+  override fun readFileChunk(path: String, offset: Double, length: Double, encoding: String, promise: Promise) {
+    ioScope.launch {
+      try {
+        val inputStream = openForReading(path);
+        inputStream.skip(offset.toLong())
+        val data = ByteArray(length.toInt())
+        inputStream.read(data);
```

Comment:
Looks like we never close the file?  Could you restructure in the form:
```kt
val data = openForReading(path).use {
  // [...]
}
```
to automatically handle closing, even if an exception is thrown?

Reply:
Will do :)